packages/salesforcedx-vscode-lwc/src/telemetry/telemetry.ts

```diff
@@ -44,15 +74,17 @@ export class TelemetryService {
     }
   }
 
-  public sendExtensionDeactivationEvent(): void {
+  public async sendExtensionDeactivationEvent(): Promise<void> {
+    await this.setupVSCodeTelemetry();
```

Comment:
This is not needed if the telemetry setup is triggered during the activation phase of the extension.